src/main/java/userInterface/Ui.java

```diff
@@ -0,0 +1,136 @@
+package userInterface;
+
+import tasks.Deadline;
+import tasks.Event;
+import tasks.Task;
+import tasks.Todo;
+
+import java.util.ArrayList;
+
+public class Ui {
+    public String printUnderscores() {
+        return "__________________________________________";
+    }
+
+    public void printHelp() {
+        String instructions = "The following is a list of commands that you can enter:\n"
+                + "1) help - For showing all commands available\n"
+                + "2) list - To display all the tasks in your list\n"
+                + "3) todo - Add a to-do task into your list (Example: todo borrow book)\n"
+                + "4) deadline - Use \" /by \" to denote deadline (Example: deadline return book /by Sunday)\n"
+                + "5) event - Use \" /at \" to denote when the event takes place (Example: event meeting /at Mon 2-4pm)\n"
+                + "6) bye - To exit from the program";
+        System.out.println(instructions);
+    }
+
+    public void printGreetingMessage() {
+        String logo = " _______    __   ___      ___  _______\n"
+                + "|__   __|  /  \\  \\  \\    /  / |   ____|\n"
+                + "   | |    / __ \\  \\  \\  /  /  |  |__\n"
+                + "   | |   / |__| \\  \\  \\/  /   |   __|\n"
+                + " __| |  /   __   \\  \\    /    |  |____\n"
+                + "|____| /___|  |___\\  \\__/     |_______|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println(printUnderscores() + "\nHello! I'm Jave" + "\nWhat can I do for you?");
+    }
+
+    public void printFarewellMessage() {
+        System.out.println(printUnderscores() + "\nBye. Hope to see you again soon!\n" + printUnderscores());
+    }
+
+    public void printList(ArrayList<Task> items) {
+        int index = 1;
+        if (items.size() == 0) {
+            System.out.println("There is nothing in your list currently");
+        } else {
+            for (Task item : items) {
+                System.out.println(index + ". " + (item).printTask());
+                index++;
+            }
+        }
+    }
+
+    public void showLoadingError() {
+        System.out.println("File tasklist.txt is not found, creating new file.......");
+    }
+
+    public void markDone(int index, ArrayList<Task> texts) {
+        if ((index <= texts.size()) && (index > 0)) {
+            (texts.get(index - 1)).setDone();
+            System.out.println(printUnderscores());
+            System.out.println("Nice! I've marked this as done:\n  " + texts.get(index - 1).printTask());
+        } else {
+            System.out.println("Invalid index!");
+        }
+    }
+
+    public void deleteTask(int index, ArrayList<Task> texts) {
+        if ((index <= texts.size()) && (index > 0)) {
```

Comment:
You may consider extracting out texts.size() into a variable called numTasks (example). Improves code readability.